src/realm/sync/instructions.hpp

```diff
@@ -670,10 +659,9 @@ struct Instruction {
         ArrayInsert = 8,
         ArrayMove = 9,
         ArrayErase = 10,
-        ArrayClear = 11,
+        Clear = 11,
         SetInsert = 12,
         SetErase = 13,
-        SetClear = 14,
```

Comment:
[q] was this present on any released branch? if so, we should make note a that `14` should be skipped if we want to introduce a new instruction type in a backwards-compatible way.